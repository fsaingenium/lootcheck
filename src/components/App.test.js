import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import App from "./App";

Enzyme.configure({ adapter: new Adapter() });

describe("App", () => {
  const app = shallow(<App />);

  it("it rendres correctly", () => {
    expect(app).toMatchSnapshot();
  });

  it("contains a wallet component", () => {
    //console.log(app.debug());
    expect(app.find("Connect(Wallet)").exists()).toBe(true);
  });
});
