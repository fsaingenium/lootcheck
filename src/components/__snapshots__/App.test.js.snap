// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App it rendres correctly 1`] = `ShallowWrapper {}`;
